bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
Maybe we could include an `en` conditional here instead? (to match the global nav)